examples/official-storybook/stories/__snapshots__/addon-knobs.stories.storyshot

```diff
@@ -0,0 +1,189 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think all these stories have been removed from next branch, should not be added back